site/index.xml

```diff
@@ -187,10 +187,10 @@ limitations under the License.
               <h2>Commercial Distribution</h2>
               <p>
                 VMware offers a <a href="https://tanzu.vmware.com/rabbitmq">range of commercial offerings for RabbitMQ</a>.
-                This includes a distribution called <a href="https://tanzu.vmware.com/services-marketplace/messaging-and-integration/rabbitmq">Tanzu RabbitMQ</a>,
-                a version that deploys in <a href="https://tanzu.vmware.com/tanzu">VMware Tanzu platform</a>,
-                and a forthcoming <a href="https://tanzu.vmware.com/content/blog/introducing-rabbitmq-for-kubernetes">version for Kubernetes</a>.
-                These distributions include all of the features of the open source version, with some additional management features. Support agreements are part of the commercial licensing.
+                This includes a distribution called <a href="https://tanzu.vmware.com/services-marketplace/messaging-and-integration/rabbitmq">Tanzu RabbitMQ</a> that deploys on Kubernetes or
```

Comment:
is this link: https://tanzu.vmware.com/services-marketplace/messaging-and-integration/rabbitmq wrong, it is pointing to: VMware Tanzu™ RabbitMQ® for VMs? Does it need to change to: https://network.tanzu.vmware.com/products/p-rabbitmq-for-kubernetes which is VMware Tanzu™ RabbitMQ® for Kubernetes